--- gnome-network-displays-0.91.0/src/cc/cc-comm.c.orig	2024-01-18 15:59:34.000000000 +0100
+++ gnome-network-displays-0.91.0/src/cc/cc-comm.c	2024-01-18 18:53:20.355505656 +0100
@@ -409,6 +409,7 @@ cc_comm_send_request (CcComm       *comm
     {
     /* CAST__CHANNEL__CAST_MESSAGE__PROTOCOL_VERSION__CASTV2_1_3 allows for binary payloads over utf8 */
     case CC_MESSAGE_TYPE_AUTH:
+    {
       ProtobufCBinaryData binary_payload;
       binary_payload.data = NULL;
       binary_payload.len = 0;
@@ -421,6 +422,7 @@ cc_comm_send_request (CcComm       *comm
                              &binary_payload,
                              NULL);
       break;
+    }
 
     default:
       cc_comm_build_message (&message,
